README.md

```diff
@@ -1,6 +1,6 @@
 <h1 align="center">
 <br>
-  <a href="https://github.com/thedaviddias/Front-End-Performance-Checklist"><img src="https://raw.githubusercontent.com/thedaviddias/Front-End-Performance-Checklist/master/images/logo-front-end-performance-checklist.jpg" alt="Front-End Performance Checklist" width="170"></a>
+  <a href="https://github.com/thedaviddias/Front-End-Performance-Checklist"><img src="https://raw.githubusercontent.com/thedaviddias/Front-End-Performance-Checklist/master/images/logo-front-end-performance-checklist.jpg" alt="Front-End Performance Checklist" width="175" height="200"></a>
```

Comment:
The logo aspect ratio in `1:1`, so this PR would present a deformed logo

![image](https://user-images.githubusercontent.com/1307927/185570095-bdeb0676-2ac5-4ee3-a6a7-64aac070e957.png)